doc/api/util.md

```diff
@@ -1593,6 +1593,40 @@ $ node negate.js --no-logfile --logfile=test.log --color --no-color
 { logfile: 'test.log', color: false }
 ```
 
+## `util.parseEnv(content)`
+
+> Stability: 1.1 - Active development
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `content` {string}
+
+The raw contents of a `.env` file.
+
+* Returns: {Object}
+
+Given an example `.env` file:
+
+```text
+HELLO=world
+```
+
+```cjs
+const { parseEnv } = require('node:util');
+
+parseEnv('HELLO=world');
+// Returns: { HELLO: 'world' }
```

Comment:
nit: maybe we could use a slightly more complicated example to better demonstrate what this is about?
```suggestion
parseEnv('HELLO=world\nHELLO=oh my\n');
// Returns: { HELLO: 'oh my' }
```